adoc/Release-Notes.adoc

```diff
@@ -103,7 +103,7 @@ endif::[]
 [id='sec.1_5_1.issue']
 ==== Known Issues
 
-* On occasion, the credhub pod may fail to start due to database migration failures; this has been spotted intermittently on {aks} and to a lesser extent, other public clouds. In these situations, manual intervention is required to track the last completed transaction in `credhub_user` database and update the `flyway schema history` table with the record of the last completed transaction. Please contact support for further instructions.
+* On occasion, the `uaa` pod may fail to start due to database migration failures. In these situations, manual intervention is required to track the last completed transaction in `uaadb` database, update the flyway schema history table with the record of the last completed transaction, then restart the migration. Please contact support for further instructions.
```

Comment:
@prabalsharma just checked - we need to replace the flyway schema history table with  `schema_version` table.

Reply:
Updated. If the wording is still incorrect, feel free to edit directly :)